cpp/BUILD

```
@@ -54,6 +54,7 @@ cc_binary(
     copts = cxxopts,
     linkopts = select({
         "@vaticle_bazel_distribution//platform:is_windows": ["ntdll.lib", "secur32.lib", "crypt32.lib", "ncrypt.lib"],
+        "@vaticle_bazel_distribution//platform:is_mac": ["-framework CoreFoundation", "-std=c++17"],
```

`-std=c++17` isn't a link options
```suggestion
        "@vaticle_bazel_distribution//platform:is_mac": ["-framework CoreFoundation"],
```